src/content/docs/get-started/index.mdx

```diff
@@ -237,168 +217,62 @@ The Code Sync app redeploys your storefront site whenever you push or merge chan
 
-1. [Install the AEM Sidekick](https://www.aem.live/docs/sidekick) from the Chrome Web Store and pin the Sidekick extension to your browser's toolbar.
-1. Open your content folder. You should now see an overlay menu bar at the bottom of your content pages.
-1. From the overlay menu, select **Sign-in**.
-1. After you are signed in, select the context menu (⫢) from the Sidekick and select **Add this project** from the options.
-1. In your content folder, select all content files, then select the **Preview** button from the Sidekick.
-1. Repeat this for all files in each folder and each subfolder.
-1. Select all the files again, but select the **Publish** button to publish these files to the CDN.
 
-These actions add your content to the internal (preview) and production (publish) Edge Delivery CDNs, where they can be downloaded and displayed in your browser for local development.
+1. Review documentation about using and installing [the AEM Sidekick](https://www.aem.live/docs/sidekick).
+1. If you are using the Config Service, you should ensure the `sidekick` property of your config with the below at minimum:
+1. If you are not using the Config Service, you need to create a file in your code repo at `tools/sidekick/config.json` with the below at minimum.
+  ```json
+  {
+    "project": "My Project",
+    "editUrlLabel": "Document Authoring",
+    "editUrlPattern": "https://da.live/edit#/{{org}}/{{site}}{{pathname}}",
+  }
+  ```
 
-</Callouts>
-
-<Aside type="note" title="Step 7 Details">
-  **The final step is only required for initial Sidekick and content set up**. Without previewing and then
-  publishing your content, the Edge Delivery CDNs remain empty, causing **404 errors** when you try
-  to run your storefront from your local development environment (our final task).
+<Aside type="note" title="">
+You can now use Sidekick to **Preview**, **Publish**, and **Edit** all your files directly from your content folder. Take a moment to select any file in your folder, select the Sidekick extension, and choose the Sidekick options to get familiar with how Sidekick works. Visit the [Sidekick documentation](https://www.aem.live/docs/sidekick) for more information on how to use Sidekick to manage your storefront content.
 </Aside>
-
-You can now use Sidekick to **Preview**, **Publish**, and **Edit** all your files directly from your Google Drive or SharePoint folder. Take a moment to select any file in your folder, select the Sidekick extension, and choose the Sidekick options to get familiar with how Sidekick works. Visit the [Sidekick documentation](https://www.aem.live/docs/sidekick) for more information on how to use Sidekick to manage your storefront content.
-
 </Task>
 
 <Task>
 
-### Set up local dev environment
+### Set up local environment
 
 <Steps>
 1. Go to your GitHub repo and select the **Code** button to copy the repo's git URL for cloning (HTTPS or SSH).
```

Comment:
Boilerplate doesn't deliver Citi signal branded site.  It's this one:
![boilerplate-site](https://github.com/user-attachments/assets/4fea80b1-28da-431f-b8c3-285135a38acb)